File: twiml/voice/sip/sip-12/sip-12.5.x.cs
using System;
using Twilio.TwiML;
using Twilio.TwiML.Voice;


class Example
{
    static void Main()
    {
        var response = new VoiceResponse();
        var dial = new Dial(sequential: true);
        dial.Sip(new Uri("sip:alice@example.com"));
        dial.Sip(new Uri("sip:bob@example.com"));
        dial.Sip(new Uri("sip:charlie@example.com"));
        response.Append(dial);

        Console.WriteLine(response.ToString());
    }
}

File: twiml/voice/sip/sip-12/sip-12.5.x.js
const VoiceResponse = require('twilio').twiml.VoiceResponse;

const response = new VoiceResponse();
const dial = response.dial({
    sequential: true
});
dial.sip('sip:alice@example.com');
dial.sip('sip:bob@example.com');
dial.sip('sip:charlie@example.com');

console.log(response.toString());
